src/redux/slices/QuranReader/sidebarNavigation.ts

```diff
@@ -0,0 +1,59 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { RootState } from 'src/redux/RootState';
+
+export type SidebarNavigation = {
+  isVisible: boolean;
+  selectedNavigationItem: string;
+};
+
+export enum NavigationItem {
+  Surah = 'surah',
+  Juz = 'juz',
+  Page = 'page',
+}
+
+export const navigationItems = [
```

Comment:
I don't think this belongs here. I think this should be moved to `src/components/QuranReader/SidebarNavigation/SidebarNavigation.tsx`. Also the value of `name` field should be localized